schema/encoding/jsonschema/array.go

```diff
@@ -1,28 +1,28 @@
 package jsonschema
 
-import (
-	"io"
-	"strconv"
+import "github.com/rs/rest-layer/schema"
 
-	"github.com/rs/rest-layer/schema"
-)
+type arrayBuilder schema.Array
 
-func encodeArray(w io.Writer, v *schema.Array) error {
-	ew := errWriter{w: w}
-
-	ew.writeString(`"type": "array"`)
+func (v arrayBuilder) JSONSchema() (map[string]interface{}, error) {
+	m := make(map[string]interface{})
+	m["type"] = "array"
```

Comment:
sure, just lazy when I removed the sizeHint for the allocation. I will get it done.